### DISCUSSION TITLE
NOW_GITHUB_DEPLOYMENT not set

### DISCUSSION THREAD

**Discussion by aericson** [vercel/vercel#4032] [General] [answered]
For some reason since yesterday `NOW_GITHUB_DEPLOYMENT` is not being set when building from the github integration. Anything changed here?

https://zeit.co/docs/v2/git-integrations/zeit-now-for-github

**Comment by lucleray** [comment, marked as answer]
Hi @aericson,

The environment should be available, do you mind sharing the deployment url that doesn't have `NOW_GITHUB_DEPLOYMENT` here or using the Feedback input on zeit.co?

**Reply by aericson** [reply]
Hello, thanks for replying. It seems like it was something temporary, triggering a redeploy now from the deployment page fixed it.
